lib/models/count_of_file_types.dart

```diff
@@ -0,0 +1,9 @@
+class CountOfFileTypes {
```

Comment:
nit: can be avoided if we decide to switch to Map<FileType, int> fileTypeToCount map.